### PR TITLE
[permissions] Use privileged-data option. Fixes JB#51951

### PR DESCRIPTION
**Pull request by spiiroin**
Privileged.permission file makes empty privileged data directory
available in sandbox. And FEATURE.permission files then make also
relevant privileged data subdirectories available.

Signed-off-by: Simo Piiroinen <simo.piiroinen@jolla.com>

### CODE REVIEW

File: rpm/sailjail-permissions.spec [hunk 1]
```diff
@@ -10,6 +10,8 @@ BuildRequires: qt5-qmake
 BuildRequires: qt5-qttools-linguist
 BuildRequires: python3-base
 
+# TBD Conflicts: firejail < 0.9.63+git3
```
Comment (spiiroin) [inline rpm/sailjail-permissions.spec:13]: This firejail dep still needs to be updated.

Btw, I'm assuming that the lack of Requires: firejail is intentional.

Comment (spiiroin) [reply rpm/sailjail-permissions.spec:13]: After consultation: not needed as none of this has yet been in any release.

**Review by Tomin1 — APPROVED** [review]
There is still comment about conflicts with firejail in commit message. Other than that LGTM!

**Comment by spiiroin** [comment]
> There is still comment about conflicts with firejail in commit message

Commit message edited.

**Comment by spiiroin** [comment]
Rebased

**Pull request merged**